pkgs/fc/agent/fc/maintenance/tests/test_reqmanager.py

```diff
@@ -240,13 +241,82 @@ def test_explicitly_deleted(connect, reqmanager):
     )
 
 
+def test_enter_maintenance(log, reqmanager, monkeypatch):
+    monkeypatch.setattr(
+        "fc.util.directory.connect", connect_mock := MagicMock()
+    )
+    monkeypatch.setattr("socket.gethostname", lambda: "host")
+
+    reqmanager.enter_maintenance()
+
+    assert log.has("enter-maintenance")
+    connect_mock().mark_node_service_status.assert_called_with("host", False)
+
+
+def test_enter_maintenance_postpone(log, reqmanager, monkeypatch):
+    monkeypatch.setattr(
+        "fc.util.directory.connect", connect_mock := MagicMock()
+    )
+    # monkeypatch.setattr("subprocess.run", run_mock := MagicMock())
```

Comment:
Is there a reason to keep this as a comment?

Reply:
no, I'll remove it in the next agent PR